src/graphics/hud_manager.rs

```diff
@@ -19,29 +29,40 @@ pub struct HUDManager {
     /// The rects that are always present on the screen
     base: Vec<RectVertexAttr>,
 
+    help_menu_data: HelpMenuData,
```

Comment:
I had removed this because there is no reason to store it. Why did you add it again ? 

Again the rule is "as little attributes as possible"